toolkit/tools/imagegen/configuration/systemconfig_test.go

```diff
@@ -214,11 +214,11 @@ func TestShouldFailParsingBadUserUID_SystemConfig(t *testing.T) {
 
```

Comment:
You should probably leave alone stuff outside of the image customizer directories.

Reply:
i just found out that user util functions like UIDIsValid and NameIsValid is called in both mic and imagegen tests, and I updated the names in those util functions which dont fit for imagegen tests